Add some information for premium connectors. (#368)
akeneo/pim-api-docs: content/guides/dam-connection/pre-requisites.md

Still not familiar with the asset family concept? You may need to read [this](/concepts/asset-manager.html#asset-family) carefully.
:::

> To create those asset families, you'll need the help of your PIM and DAM users. By analyzing together the assets already stored in your DAM, you'll find the perfect set of asset families with the right attribute structure to optimize your asset representation in the PIM.

Reviewer: ```suggestion
To create those asset families, you'll need the help of your PIM and DAM users. By cross-analyzing the assets already stored in your DAM, you'll find the perfect set of asset families with the right attribute structure to optimize your asset representation in the PIM.
```

### Create the structure in the Asset Manager

Once you have a better idea of the modelization of your asset families, you can create them.  

> You can either do this creation in the PIM, or you can use a script leveraging our API. You can find an example of this script in our [DAM connector skeleton](introduction.html#the-skeleton).

Reviewer: ```suggestion
You can either do it in the PIM, or you can use a script leveraging our API. You can find an example of this script in our [DAM connector skeleton](introduction.html#the-skeleton).
```

To create an asset family with its attributes and its product link rules, you'll need to follow those steps:
1. Create your asset family with its code and labels.  
2. Add asset attributes to your family.
3. In your asset family properties, define which media link or media file attribute will be your [main media attribute](/concepts/asset-manager.html#asset-family).   
4. Add the product link rules to your asset family.

::: warning

> When creating your `media link` or `media file` attributes, please check that in these attributes properties that you have defined its `type` (image, video, pdf, youtube, other...).

Reviewer: ```suggestion
When creating your `media link` or `media file` attributes, please check that you have defined their `type` (image, video, pdf, youtube, other...) in the attributes properties.
```

### Create the structure in the Asset Manager

Once you have a better idea of the modelization of your asset families, you can create them.  

> You create them via PIM UI or its API.

Reviewer 2: ```suggestion
You can create them in the PIM UI or via the API.
```

> If you want to do this via API, we advise you to use a script using the API. You can find an example of this script in our [DAM connector skeleton](introduction.html#the-skeleton).

Reviewer 2: ```suggestion
You can either do this creation in the PIM, or you can use a script leveraging our API. You can find an example of this script in our [DAM connector skeleton](introduction.html#the-skeleton).
```

To create an asset family with its attributes and its product link rules, you'll need to follow those steps:
1. Create your asset family with its code and labels.  
2. Add asset attributes to your family.

> 3. In your asset family properties, define which of your media attributes is the main media attribute.   

Reviewer 2: ```suggestion
3. In your asset family properties, define which media link or media file attribute will be your main media attribute.   
```
